### PR TITLE
Added new term for proteomics

### PR DESCRIPTION
**Pull request by cw-dvr8**
Added a proteomics term I'd missed in the last PR.

### CODE REVIEW

File: reference_definitions_schemas/new_schema_refs.json [hunk 1]
```diff
@@ -34,6 +34,10 @@
       "description": "Chromatin amount",
       "type": "number"
     },
+    "controlType": {
+      "description": "Control samples suitable for normalization and batch correction",
```
Comment (kelshmo) [inline reference_definitions_schemas/new_schema_refs.json]: I don't know how to interpret this definition and I'm further confused by numbers as the allowable type.

Comment (cw-dvr8) [reply reference_definitions_schemas/new_schema_refs.json]: @kelshmo I've corrected the type to string.

I don't know how to interpret the definition either, it was copied directly from the AMP-AD TMTquantitation assay.  @amapeters , @karawoo , can you clarify?

Comment (kelshmo) [reply reference_definitions_schemas/new_schema_refs.json]: @amapeters please comment on specific controls used e.g. GIS.

**Comment by amapeters** [comment]
Value for controlType = GIS. I do have an official definition for it, but this is what was provided by the Emory proteomics core "Global Internal Standard (GIS). A specific control for TMT multi-batch proteomic experiments. Is typically a representative mixture of many or all samples from the cohort, and can be considered an average overall protein expression profile for the cohort. Used for normalization and batch correction."

**Comment by cw-dvr8** [comment]
@amapeters Should this be a controlled vocabulary column, or should we allow contributors to enter whatever they want?  If it's controlled vocabulary, what are the values that belong in here?

**Comment by amapeters** [comment]
Can we use 'GIS' as value for that key, but not make it required ?

**Comment by cw-dvr8** [comment]
@amapeters do you mean that we not make the key required, or the value "GIS" not required, i.e. they can either choose "GIS" or enter something else?

@kelshmo how would that play with the dccvalidator?

**Comment by amapeters** [comment]
Choose "GIS" or enter something else. Otherwise we can make the value required. People will just need to inform us if we need to add additional control values

**Comment by cw-dvr8** [comment]
I can do that with the schema because I'm not using it to validate, but I don't know if the dccvalidator would allow it.  Will wait for @kelshmo to weigh in.

**Comment by kelshmo** [comment]
GIS will be an enumerated value that is not required. If a contributor needs to use another value, they will need to request it is added to the schema which is currently communicated via email.

File: reference_definitions_schemas/new_schema_refs.json [hunk 2]
```diff
@@ -34,6 +34,10 @@
...
+      "type": "string"
```
Comment (kelshmo) [inline reference_definitions_schemas/new_schema_refs.json]: Add GIS as a `const`.

Comment (cw-dvr8) [reply reference_definitions_schemas/new_schema_refs.json]: Done.

**Pull request merged**